From 5ac6717b0baccc7c75fba24fd2657ec51757deef Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 19 Feb 2020 10:57:34 +0100
Subject: [PATCH 1235/1852] ARM: zynq: Fix addresses in partition definitions

commit 55a7aa73cd38939ba4c3c0bd674f34ce38ffc203 from
https://github.com/Xilinx/linux-xlnx.git

Node name should be <name>@<address> which is not how partitions are
described.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm/boot/dts/zynq-zc702.dts       | 10 +++++-----
 arch/arm/boot/dts/zynq-zc706.dts       | 10 +++++-----
 arch/arm/boot/dts/zynq-zc770-xm010.dts | 10 +++++-----
 arch/arm/boot/dts/zynq-zc770-xm011.dts | 10 +++++-----
 arch/arm/boot/dts/zynq-zc770-xm012.dts | 10 +++++-----
 arch/arm/boot/dts/zynq-zc770-xm013.dts | 10 +++++-----
 arch/arm/boot/dts/zynq-zed.dts         | 10 +++++-----
 7 files changed, 35 insertions(+), 35 deletions(-)

diff --git a/arch/arm/boot/dts/zynq-zc702.dts b/arch/arm/boot/dts/zynq-zc702.dts
index c9940fb366ce..e3c64ca487ed 100644
--- a/arch/arm/boot/dts/zynq-zc702.dts
+++ b/arch/arm/boot/dts/zynq-zc702.dts
@@ -416,23 +416,23 @@
 		spi-max-frequency = <50000000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		partition@qspi-fsbl-uboot {
+		partition@0 {
 			label = "qspi-fsbl-uboot";
 			reg = <0x0 0x100000>;
 		};
-		partition@qspi-linux {
+		partition@100000 {
 			label = "qspi-linux";
 			reg = <0x100000 0x500000>;
 		};
-		partition@qspi-device-tree {
+		partition@600000 {
 			label = "qspi-device-tree";
 			reg = <0x600000 0x20000>;
 		};
-		partition@qspi-rootfs {
+		partition@620000 {
 			label = "qspi-rootfs";
 			reg = <0x620000 0x5E0000>;
 		};
-		partition@qspi-bitstream {
+		partition@c00000 {
 			label = "qspi-bitstream";
 			reg = <0xC00000 0x400000>;
 		};
diff --git a/arch/arm/boot/dts/zynq-zc706.dts b/arch/arm/boot/dts/zynq-zc706.dts
index 1a1b03a4223d..5c9f14d4dd46 100644
--- a/arch/arm/boot/dts/zynq-zc706.dts
+++ b/arch/arm/boot/dts/zynq-zc706.dts
@@ -320,23 +320,23 @@
 		spi-max-frequency = <50000000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		partition@qspi-fsbl-uboot {
+		partition@0 {
 			label = "qspi-fsbl-uboot";
 			reg = <0x0 0x100000>;
 		};
-		partition@qspi-linux {
+		partition@100000 {
 			label = "qspi-linux";
 			reg = <0x100000 0x500000>;
 		};
-		partition@qspi-device-tree {
+		partition@600000 {
 			label = "qspi-device-tree";
 			reg = <0x600000 0x20000>;
 		};
-		partition@qspi-rootfs {
+		partition@620000 {
 			label = "qspi-rootfs";
 			reg = <0x620000 0x5E0000>;
 		};
-		partition@qspi-bitstream {
+		partition@c00000 {
 			label = "qspi-bitstream";
 			reg = <0xC00000 0x400000>;
 		};
diff --git a/arch/arm/boot/dts/zynq-zc770-xm010.dts b/arch/arm/boot/dts/zynq-zc770-xm010.dts
index 12d13588402e..4e59d0c2ae91 100644
--- a/arch/arm/boot/dts/zynq-zc770-xm010.dts
+++ b/arch/arm/boot/dts/zynq-zc770-xm010.dts
@@ -72,23 +72,23 @@
 		spi-max-frequency = <50000000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		partition@qspi-fsbl-uboot {
+		partition@0 {
 			label = "qspi-fsbl-uboot";
 			reg = <0x0 0x100000>;
 		};
-		partition@qspi-linux {
+		partition@100000 {
 			label = "qspi-linux";
 			reg = <0x100000 0x500000>;
 		};
-		partition@qspi-device-tree {
+		partition@600000 {
 			label = "qspi-device-tree";
 			reg = <0x600000 0x20000>;
 		};
-		partition@qspi-rootfs {
+		partition@620000 {
 			label = "qspi-rootfs";
 			reg = <0x620000 0x5E0000>;
 		};
-		partition@qspi-bitstream {
+		partition@c00000 {
 			label = "qspi-bitstream";
 			reg = <0xC00000 0x400000>;
 		};
diff --git a/arch/arm/boot/dts/zynq-zc770-xm011.dts b/arch/arm/boot/dts/zynq-zc770-xm011.dts
index 142e7263a177..76e9ecc0d01e 100644
--- a/arch/arm/boot/dts/zynq-zc770-xm011.dts
+++ b/arch/arm/boot/dts/zynq-zc770-xm011.dts
@@ -57,23 +57,23 @@
 	arm,nand-cycle-t5 = <0x2>;
 	arm,nand-cycle-t6 = <0x4>;
 
-	partition@nand-fsbl-uboot {
+	partition@0 {
 		label = "nand-fsbl-uboot";
 		reg = <0x0 0x100000>;
 	};
-	partition@nand-linux {
+	partition@100000 {
 		label = "nand-linux";
 		reg = <0x100000 0x500000>;
 	};
-	partition@nand-device-tree {
+	partition@600000 {
 		label = "nand-device-tree";
 		reg = <0x600000 0x20000>;
 	};
-	partition@nand-rootfs {
+	partition@620000 {
 		label = "nand-rootfs";
 		reg = <0x620000 0x5E0000>;
 	};
-	partition@nand-bitstream {
+	partition@c00000 {
 		label = "nand-bitstream";
 		reg = <0xC00000 0x400000>;
 	};
diff --git a/arch/arm/boot/dts/zynq-zc770-xm012.dts b/arch/arm/boot/dts/zynq-zc770-xm012.dts
index e0e5980200cb..849230ee4072 100644
--- a/arch/arm/boot/dts/zynq-zc770-xm012.dts
+++ b/arch/arm/boot/dts/zynq-zc770-xm012.dts
@@ -63,23 +63,23 @@
 	xlnx,sram-cycle-t4 = <0x3>;
 	xlnx,sram-cycle-t5 = <0x3>;
 	xlnx,sram-cycle-t6 = <0x2>;
-	partition@nor-fsbl-uboot {
+	partition@0 {
 		label = "nor-fsbl-uboot";
 		reg = <0x0 0x100000>;
 	};
-	partition@nor-linux {
+	partition@100000 {
 		label = "nor-linux";
 		reg = <0x100000 0x500000>;
 	};
-	partition@nor-device-tree {
+	partition@600000 {
 		label = "nor-device-tree";
 		reg = <0x600000 0x20000>;
 	};
-	partition@nor-rootfs {
+	partition@620000 {
 		label = "nor-rootfs";
 		reg = <0x620000 0x5E0000>;
 	};
-	partition@nor-bitstream {
+	partition@c00000 {
 		label = "nor-bitstream";
 		reg = <0xC00000 0x400000>;
 	};
diff --git a/arch/arm/boot/dts/zynq-zc770-xm013.dts b/arch/arm/boot/dts/zynq-zc770-xm013.dts
index d6c5a7b1baf2..e2c74675decb 100644
--- a/arch/arm/boot/dts/zynq-zc770-xm013.dts
+++ b/arch/arm/boot/dts/zynq-zc770-xm013.dts
@@ -71,23 +71,23 @@
 		spi-max-frequency = <50000000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		partition@qspi-fsbl-uboot {
+		partition@0 {
 			label = "qspi-fsbl-uboot";
 			reg = <0x0 0x100000>;
 		};
-		partition@qspi-linux {
+		partition@100000 {
 			label = "qspi-linux";
 			reg = <0x100000 0x500000>;
 		};
-		partition@qspi-device-tree {
+		partition@600000 {
 			label = "qspi-device-tree";
 			reg = <0x600000 0x20000>;
 		};
-		partition@qspi-rootfs {
+		partition@620000 {
 			label = "qspi-rootfs";
 			reg = <0x620000 0x5E0000>;
 		};
-		partition@qspi-bitstream {
+		partition@c00000 {
 			label = "qspi-bitstream";
 			reg = <0xC00000 0x400000>;
 		};
diff --git a/arch/arm/boot/dts/zynq-zed.dts b/arch/arm/boot/dts/zynq-zed.dts
index b9df9b30fc8d..2d531a6ea2ca 100644
--- a/arch/arm/boot/dts/zynq-zed.dts
+++ b/arch/arm/boot/dts/zynq-zed.dts
@@ -65,23 +65,23 @@
 		m25p,fast-read;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		partition@qspi-fsbl-uboot {
+		partition@0 {
 			label = "qspi-fsbl-uboot";
 			reg = <0x0 0x100000>;
 		};
-		partition@qspi-linux {
+		partition@100000 {
 			label = "qspi-linux";
 			reg = <0x100000 0x500000>;
 		};
-		partition@qspi-device-tree {
+		partition@600000 {
 			label = "qspi-device-tree";
 			reg = <0x600000 0x20000>;
 		};
-		partition@qspi-rootfs {
+		partition@620000 {
 			label = "qspi-rootfs";
 			reg = <0x620000 0x5E0000>;
 		};
-		partition@qspi-bitstream {
+		partition@c00000 {
 			label = "qspi-bitstream";
 			reg = <0xC00000 0x400000>;
 		};
-- 
2.31.1

